### ISSUE TITLE
Caster offhands missing in the deadly weapons vendor

### ISSUE THREAD

**Issue by NitzanPeer** [gamefreedomgit/Frostmourne#1953]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (https://www.wowhead.com/wotlk)

**Description:**
Zom Bocom, the deadly weapons vendor, offers wands, staves and main hand but not off hands for casters.
It should offer these AFAIK:
[Deadly Gladiator's Endgame](https://www.wowhead.com/item=42525/deadly-gladiators-endgame)
[Deadly Gladiator's Grimoire](https://www.wowhead.com/item=42537/deadly-gladiators-grimoire)
[Deadly Gladiator's Reprieve](https://www.wowhead.com/item=42531/deadly-gladiators-reprieve)

**How to reproduce:**

**How it should work:**

**Database links:**

**Comment by Taicex** [comment]
Same as sigils,relics,librarms and totems,they are also missing

**Comment by NitzanPeer** [comment]
Riztazz I don't know about what Taicex said (not sure if the items he mentioned should be sold with no rating restriction, for honor alone) but if we go back to the OP, the grimoires I posted the links to do not exist in the vendor (Zom Bocom), it's not about class. I see no logical reason for us to be able to buy a deadly staff (which is the equivalent of MH + OH) and a deadly mainhand but not a deadly offhand to go with it. If that's how things were on wotlk retail I can't find anything to support that.

**Comment by Riztazz** [comment]
Hence why i've asked what kind of class are you. I can see the item just fine.

**Comment by NitzanPeer** [comment]
I play lock, mage, priest, dk, all see the same items it seems, none see the caster offhands.

**Comment by GM-Nightmare** [comment]
@Riztazz it's not visible to players , even to me while i have gm on/off

**Comment by Riztazz** [comment]
Alright, i'll get a copy of database from the server and see what's up, i see all of them on my local server :/

**Comment by Riztazz** [comment]
Fixed, will close once it's live

**Comment by TheGreatTesla** [comment]
I'm not sure if it has been noted yet, but Zom Bocom currently only sells Emblem of Ascension pants, as the entirety of his normal Deadly Gladiator vendor pool seems to have completely disappeared. I made a separate issue for this over here:

https://github.com/SallyWhitemane/Mograine/issues/1406

Not sure if this classifies as part of this issue, or a separate one.

**Comment by Riztazz** [comment]
Delete your cache, the vendors have been changed recently, you're looking at wrong vendor

**Comment by NitzanPeer** [comment]
Deleted the cache folder and can still confirm what TheGreatTesla said, In Zom Bocom's place now stands a human farmer which sells asc legs. Zom Bocom stands in a different place and sells hateful main pieces now. Deadly weapons are nowhere to be found.

**Comment by NitzanPeer** [comment]
Since deadly weapons are last season's gear they shouldn't be locked behind rating.

@Riztazz I bet you deal with lots of stuff so I just want to sum up what happened in the last weeks in case it helps solve this issue once and for all: 
The deadly weapons vendor, Zom Bocom, was implemented and it sold what it should have, for an appropriate price (honor), with no rating requirement. The only problem was caster offhands were missing (another guy claimed sigils, relics, librams and totems are all missing as well, I don't know about that). 

In the latest update (I believe)  Zom Bocom changed it's stock and location and is now selling hateful main parts (which were already sold by another npc prior to the update, in that exact location). In Zom Bocom's original location now stands a new vendor which sells asc pve legs (which shouldn't be there as this is a pvp vendor zone) and finally - no one sells deadly weapons anymore.
So in short what I think should happen is:
1. New pve legs vendor should be removed or relocated to it's proper location.
2. Location changes should be reverted and the old npc which sold hateful main pieces should do so again.
3. Zom Bocom should return to it's original position (where the pve vendor now stands) and should sell the entire deadly stock it should sell, including caster offhands etc, for honor, with no rating requirements.

**Comment by Riztazz** [comment]
That's something you gotta talk to with Radeghost, they made the position choices and so on.
On 3, again, after talking with radeghost you will require some AP and tons of honor for deadly weapons

**Comment by NitzanPeer** [comment]
@Riztazz I see. When will the weapons be available again then? this is important for the pvp community.
@RadeghostWM Can you help? this pve vendor is found in the pvp zone and sells legs which are already sold in the main asc emblems vendor + the deadly weapons previously sold are nowhere to be found. Seems like a bug to me.

![bugybug](https://github.com/user-attachments/assets/58c6bec9-f4eb-4069-9843-ef74a1b7d5d6)

**Comment by Riztazz** [comment]
They will be available at Xazi Smolderpipe, fixed, will close once it's live
